content/webapp/components/MediaObjectBase/MediaObjectBase.test.tsx

```diff
@@ -19,31 +23,44 @@ const getBaseTitleClass = number => {
 
 const mockOnClick = jest.fn();
 
-const grid2 = grid({ s: 2, m: 2, l: 2, xl: 2 });
-const grid3 = grid({ s: 3, m: 3, l: 3, xl: 3 });
-const grid9 = grid({ s: 9, m: 9, l: 9, xl: 9 });
-const grid10 = grid({ s: 10, m: 10, l: 10, xl: 10 });
-const grid12 = grid({ s: 12, m: 12, l: 12, xl: 12 });
+const grid2: SizeMap = {
+  s: [2],
+  m: [2],
+  l: [2],
+  xl: [2],
+};
+const grid10: SizeMap = {
+  s: [10],
+  m: [10],
+  l: [10],
+  xl: [10],
+};
+const grid12: SizeMap = {
+  s: [12],
+  m: [12],
+  l: [12],
+  xl: [12],
+};
```

Comment:
Do these have to be recreated in the test or could they be imported from `common/views/components/Layout/index.tsx` ? (not a blocker, not for this ticket as pre-existing and anyway I do think we have to recreate stuff for tests...)